### PR TITLE
Fix Alignment Issues for Image-based Radio Buttons in Multipage Forms

### PR DESCRIPTION
**Pull request by shervElmi**
This PR addresses the misalignment of radio buttons displayed as images in centered multipage forms. It aims for a consistent layout across different screen sizes.

## Related Issue:
[Pro Issue - #3626](https://github.com/Strategy11/formidable-pro/issues/3626)

## QA URL
https://qa.formidableforms.com/sherv6/wp-admin/admin-ajax.php?action=frm_forms_preview&form=image-basedradiobuttons&theme=1

## Testing Instructions:
1. Activate "Formidable Pro" plugin.
2. Navigate to "WP Admin > Formidable > Forms".
3. Create a form with a multipage setup and make sure the form is centered on the page.
4. Add a "Radio Buttons" field and change the display format option to images.
5. Ensure everything is centered and aligned correctly with the radio buttons.

### Before
image

### After
image

### CODE REVIEW

**Comment by garretlaxton** [comment]
Should these images be centered on this page?
image

**Comment by shervElmi** [comment]
@garretlaxton, I've read the entire ticket and noticed that the support team provided custom CSS to the user for centering forms and buttons. You can check the custom CSS styles I've added in the QA URL provided in the description.

**Comment by garretlaxton** [comment]
Thanks, one last question... should regular radio buttons be affected by this?
image

**Comment by shervElmi** [comment]
No, @garretlaxton, the reason the radio buttons are centered in your image is due to custom CSS!

**Review by garretlaxton — APPROVED** [review]
I missed that, thanks! This looks good.

**Comment by shervElmi** [comment]
Thanks @garretlaxton, for all the reviews!

**Comment by Crabcyborg** [comment]
@stephywells Do you think there would be any issues with making radio buttons/checkboxes displayed as images 20 pixels larger?

There's a `padding-left: 20px` rule that's making them smaller right now, that causes issue with centering. It seems unintentional, but removing it means all of the images will become larger, showing an additional 20px of width.

That shouldn't be a problem right? Or would we want to reduce the width of the image options by 20px?

**Comment by shervElmi** [comment]
@Crabcyborg and @stephywells, I also wrote CSS with a more specific selector for these images to prevent any issues. Just wanted to let you know this :)

**Review by Crabcyborg — CHANGES_REQUESTED** [review]
@shervElmi There aren't any image-based classes used here. This update breaks regular radio buttons at the moment.

Screen Shot 2023-10-23 at 5 24 31 PM

Also, can we put this update in Pro as the "images" options is a Pro feature?

**Comment by shervElmi** [comment]
Ah sorry! my mistake @Crabcyborg!

**Comment by stephywells** [comment]
> Do you think there would be any issues with making radio buttons/checkboxes displayed as images 20 pixels larger?
> 
> There's a `padding-left: 20px` rule that's making them smaller right now, that causes issue with centering. It seems unintentional, but removing it means all of the images will become larger, showing an additional 20px of width.
> 
> That shouldn't be a problem right? Or would we want to reduce the width of the image options by 20px?

Is it possible to see a before and after screenshot? I'm not entirely sure what you're referring to. Does this remove the spacing in examples like the Formidable logo here?
https://github.com/Strategy11/formidable-forms/pull/1338#issuecomment-1769072760

**Comment by Crabcyborg** [comment]
@stephywells Sherv has before and after screenshots in this pull request description.

The box containing the image options for radio buttons was basically padded with an extra 20px of white space to the left, exposing less of the image.

**Comment by stephywells** [comment]
Oh! That's what you were talking about. 
@shervElmi can you also share before and after for a wider screen?

**Comment by Crabcyborg** [comment]
@shervElmi When you get the chance, could you follow-up with Steph's question from last week?

I'm going to close this as we'll want to do the update in Pro, not here.

**Comment by shervElmi** [comment]
@Crabcyborg, I'll send a screenshot tomorrow as I don't have access to a large screen right now.

**Comment by shervElmi** [comment]
@Crabcyborg, I don't know why, but all my QA URLs are broken because the Formidable Forms plugins are deleted and there is only a stable version available.

@stephywells, here is the result on the bigger screen:

<kbd>
image
</kbd>

**Comment by Crabcyborg** [comment]
Sorry @shervElmi, sometimes the number of versions gets out of hand and requires some clean up.

Yours may have been deleted be mistake.

**Comment by stephywells** [comment]
> Do you think there would be any issues with making radio buttons/checkboxes displayed as images 20 pixels larger?
> 
> There's a `padding-left: 20px` rule that's making them smaller right now, that causes issue with centering. It seems unintentional, but removing it means all of the images will become larger, showing an additional 20px of width.
> 
> That shouldn't be a problem right? Or would we want to reduce the width of the image options by 20px?

Thanks for the screenshots!

@Crabcyborg @shervElmi I don't see an issue with that extra 20px.
